library/func_misc.py

```diff
@@ -221,3 +222,200 @@ def Call(self, pos_args, named_args):
             return value.List([val])  # use val to avoid needlessly copy
 
         return value.List([])
+
+
+class _Type(vm._Callable):
+
+    def __init__(self):
+        # type: () -> None
+        pass
+
+    def Call(self, pos_args, named_args):
+        # type: (List[value_t], Dict[str, value_t]) -> value_t
+
+        r = typed_args.Reader(pos_args, named_args)
+        val = r.PosValue()
+        r.Done()
+
+        tname = ValType(val)
```

Comment:
We are a little inconsistent about this, but generally the default should be `import ui` and then `ui.ValType()`